src/main/kotlin/blackjack/model/Suit.kt

```diff
@@ -0,0 +1,8 @@
+package blackjack.model
+
+enum class Suit(val symbol: String) {
+    SPADE("♠"),
+    HEART("♥"),
+    CLUB("♣"),
+    DIAMOND("♦"),
```

Comment:
I see visual effects that makes the game even more fun 👍 
But here's my thought:

Since the visual effects are used in console based output, I think they belong to the `OutputView`.
When using different output strategy other than console, we could use other visual component rather than a String. Same applies to `Rank`. How does this sound?